kinder/doc/getting-started.md

```diff
@@ -6,7 +6,7 @@ See [Prepare for tests](prepare-for-tests.md) for how to create a node-image for
 
 ```bash
 # create a cluster with eventually some worker node
-kinder create cluster --image kindest/node:vX --workers-nodes 0
+kinder create cluster --image kindest/node:vX --worker-nodes 0
```

Comment:
/assign @fabriziopandini 
should the comment above say:
> # create a cluster without worker nodes

Reply:
Yes, please fix the comment as well